oam/install_scripts/columnstore-pre-uninstall.in

```diff
@@ -22,29 +21,32 @@ else
     kill $(pidof $PROGS) > /dev/null
     sleep 3
     kill -9 $(pidof $PROGS) > /dev/null
+    clearShm  > /dev/null 2>&1
 fi
 
-rm -f /etc/profile.d/columnstoreAlias.sh
+if [ -n "$(pgrep -x ProcMon)" ] || [ -n "$(pgrep -x ProcMgr)" ];then
+   # Old system must be running, kill ProcMon/ProcMgr 
+    pkill ProcMon
+    pkill ProcMgr
+fi
 
+rm -f /etc/profile.d/columnstoreAlias.sh
 rm -f /var/log/mariadb/columnstore/activeAlarms > /dev/null 2>&1
 rm -f /var/log/mariadb/columnstore/*.log1 > /dev/null 2>&1
 
-# delete Mariab Columnstore shared memory segments
-clearShm  > /dev/null 2>&1
-
 #get temp directory
 tmpDir=`mcsGetConfig SystemConfig SystemTempFileDir`
 if [[ $tmpDir = *"tmp" ]]; then
 	# delete tmp files
 	rm -rf $tmpDir/*
 fi
+
 rm -f /var/lib/columnstore/local/*.columnstore
 rm -rf /var/lib/columnstore/local/etc/
 rm -f /var/lib/columnstore/data/bulk/tmp/job/* >/dev/null 2>&1
 rm -f /var/lib/columnstore/local/moveDbrootTransactionLog
 rm -rf /var/log/mariadb/columnstore/
 rm -rf /tmp/columnstore_tmp_files
-rm -rf /usr/share/columnstore
```

Comment:
No need for this. Will already be cleaned up during yum remove mariadb-columnstore